fixit/rules/no_redundant_lambda.py

```diff
@@ -0,0 +1,76 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import libcst as cst
+import libcst.matchers as m
+from libcst.helpers import get_full_name_for_node
+
+from fixit import CstLintRule, InvalidTestCase as Invalid, ValidTestCase as Valid
+
+
+UNNECESSARY_LAMBDA: str = (
+    "The lambda that is wrapping {function} is redundant. "
+    + "It can unwrapped safely and used purely."
+)
+
+
+class NoRedundantLambdaFunction(CstLintRule):
+
```

Comment:
We use Rule as the naming convention.
https://fixit.readthedocs.io/en/latest/build_a_lint_rule.html#Pick-a-Good-Name
```suggestion
class NoRedundantLambdaRule(CstLintRule):

```

Reply:
Oh completely missed that tutorial! Thanks